src/Core/Grid/Query/CmsPageQueryBuilder.php

```diff
@@ -199,4 +200,19 @@ private function getModifiedPositionFilter($positionFilterValue)
 
         return $reducedByOneFilterValue;
     }
+
+    /**
+     * @return string Query to get is_same property
+     */
+    public static function getIsSameQuery($dbPrefix)
```

Comment:
```suggestion
    public static function getIsSameQuery(string $dbPrefix): string
```

Reply:
OK, this too is missing almost everywhere.